content/en/ecosystem/adopters.md

```diff
@@ -1,39 +1,24 @@
 ---
 title: Adopters
-description: Organizations that have adopted OpenTelemetry
+description: End-user organizations that have adopted OpenTelemetry
 # prettier-ignore
 spelling: cSpell:ignore Dapr Datenrettungsdienste Globale Logicmonitor Logz Wandera Zocdoc
 # All spelling entries must be on a single line
 ---
 
-A non-exhaustive, alphabetized list of organizations that have adopted
-OpenTelemetry.
+A non-exhaustive, alphabetized list of [end user organizations](https://community.cncf.io/end-user-community/) that have adopted OpenTelemetry. Note that to be an end user for OpenTelemetry, an organization cannot sell an Observability product.
```

Comment:
```suggestion
A non-exhaustive, alphabetized list of [end-user organizations](https://community.cncf.io/end-user-community/) that have adopted OpenTelemetry. Also see [Vendors](../vendors/).
```

Reply:
I've reapplied these suggestions to the updated file, and so marking as resolved.
PTAL